### PR TITLE
updates gpcheck parameters

### PR DESCRIPTION
**Pull request by ChrisYuan**
I install the gpdb and perform check, there are some mismatched parameters between documentation and gpcheck.conf, I fixed them .

### CODE REVIEW

**Comment by d** [comment]
Please don't use merge commits in your pull request and rebase instead

File: gpMgmt/bin/gpseginstall [hunk 1]
```diff
@@ -311,7 +311,7 @@ def checkUsernames():
     logger.info("check for user %s on cluster" % options.user)
 
     try:
-        for k in uniqueseghosts.keys():
+        for k in uniqueseghosts.keys():pqpgthen
```
Comment (danielgustafsson) [inline gpMgmt/bin/gpseginstall]: I assume this wasn't intentional?

File: gpdb-doc/dita/install_guide/prep_os_install_gpdb.xml [hunk 2]
```diff
@@ -198,6 +198,7 @@ kernel.msgmnb = 65536
 kernel.msgmax = 65536
 kernel.msgmni = 2048
 net.ipv4.tcp_syncookies = 1
+net.ipv4.ip_forward = 0
```
Comment (tylarb) [inline gpdb-doc/dita/install_guide/prep_os_install_gpdb.xml]: I don't see this ip_forward setting in any of our docs currently - what is the reasoning for adding this in this PR? 

It seems like net.ipv4.ip_forward  could be worth documenting (although some research indicates ip_forward is disabled by default on most distros) , but it does not appear relevant to the other content in this PR. 

(confirmed ip_forward is disabled by default on new install RHEL 6/7)

~~~
# cat /proc/sys/net/ipv4/ip_forward
0
~~~

**Comment by ChrisYuan** [comment]
Thanks for all of you , I will follow @d and @danielgustafsson 's guide to pull a rebase branch in the future, sorry for delay.

**Pull request closed without merging**